.settings/com.google.appengine.eclipse.core.prefs

```diff
@@ -1,5 +1,5 @@
 eclipse.preferences.version=1
-filesCopiedToWebInfLib=appengine-api-1.0-sdk-1.9.27.jar|appengine-api-labs.jar|appengine-endpoints-deps.jar|appengine-endpoints.jar|appengine-jsr107cache-1.9.27.jar|datanucleus-appengine-1.0.10.final.jar|datanucleus-core-1.1.5.jar|datanucleus-jpa-1.1.5.jar|geronimo-jpa_3.0_spec-1.1.1.jar|geronimo-jta_1.1_spec-1.1.1.jar|jdo2-api-2.3-eb.jar|jsr107cache-1.1.jar
+filesCopiedToWebInfLib=appengine-api-1.0-sdk-1.9.50.jar|appengine-api-labs.jar|appengine-endpoints-deps.jar|appengine-endpoints.jar|appengine-jsr107cache-1.9.50.jar|datanucleus-appengine-1.0.10.final.jar|datanucleus-core-1.1.5.jar|datanucleus-jpa-1.1.5.jar|geronimo-jpa_3.0_spec-1.1.1.jar|geronimo-jta_1.1_spec-1.1.1.jar|jdo2-api-2.3-eb.jar|jsr107cache-1.1.jar
 gaeDatanucleusVersion=v1
```

Comment:
>It came automatically

I didn't get you.

> I am using this version of appengine, maybe thats why. Should i use appengine-api-1.0-sdk-1.9.27

You should use 1.9.27. It is mentioned [here](https://github.com/TEAMMATES/teammates/blob/master/docs/settingUp.md#step-3-set-up-project-specific-settings-and-dependencies). Actually if you have just followed the step 3 it will download the 1.9.27 only. The command `./gradlew appengineDownloadSdk` will do that for you.